src/components/Nav/Nav.js

```diff
@@ -108,4 +132,10 @@ class Nav extends Component {
   }
 }
 
-export default Nav;
+const mapStateToProps = (state) => {
+  return { loginToken: state.loginToken, currentPage: state.currentPage };
+};
+
+export default connect(mapStateToProps, { logoutHandler, setCurrentPage })(
```

Comment:
```suggestion
export default connect(mapStateToProps, { setLogout, setCurrentPage })(
```
redux 액션 생성함수라는 걸 바로 알아 볼 수 있도록 함수 이름을 통일시켜 주면 좋을 것 같습니다.

Reply:
수정했습니다~!